### DISCUSSION TITLE
Integeration with Zabbix

### DISCUSSION THREAD

**Discussion by mahdavi3d** [TechnitiumSoftware/DnsServer#1132] [Ideas]
I wish there was a way to be able to monitor Technitium DnsServer Trough Zabbix dashboard with similar features as local dashboard. Before you ask, the Url widget in Zabbix doesn't show the page for me.

**Comment by ShreyasZare** [comment]
Thanks for the post. Its not feasible to develop integration for Zabbix. I would suggest to ask them to provide support for it.

**Comment by zbalkan** [comment]
It's in my todo list, so I can give you a hint @mahdavi3d. You can use [Get Stats API](https://github.com/TechnitiumSoftware/DnsServer/blob/master/APIDOCS.md#get-stats) with [Zabbix HTTP agent](https://www.zabbix.com/documentation/current/en/manual/config/items/itemtypes/http).
